algorithms/indexing/basis_vector_search/fft1d.py

```diff
@@ -16,13 +16,32 @@
 
 
 class FFT1D(Strategy):
-    """Basis vector search using a 1D FFT.
+    """
+    Basis vector search using 1D FFTs in reciprocal space.
+
+    A set of dimensionless radial unit vectors, typically ~7000 in total, is chosen
+    so that they are roughly evenly spaced in solid angle over a hemisphere. The
+    reciprocal space displacements of the measured spot centroids are then projected
+    onto each of these radial vectors in turn (that is, we calculate the scalar
+    product of each displacement with each unit vector).  A 1D FFT of the linear
+    density of projected spot positions is performed along each direction.
+    Aggregating the results of all the transforms, the three shortest non-collinear
+    wave vectors with the greatest spectral weight correspond to the basis vectors of
+    the direct lattice.
 
     See:
         Steller, I., Bolotovsky, R. & Rossmann, M. G. (1997). J. Appl. Cryst. 30, 1036-1040.
         Sauter, N. K., Grosse-Kunstleve, R. W. & Adams, P. D. (2004). J. Appl. Cryst. 37, 399-409.
     """
 
+    phil_help = (
+        " 'Search for the basis vectors of the direct lattice by performing a series "
```

Comment:
As well as being untidy, I'm not convinced the extra whitespace padding and single quotes are even necessary - the following still appears to work:
```
libtbx.phil.parse("""
%s
.expert_level=1
.help=%s
{}
""" % (entry_point.name, entry_point.load().phil_help[2:-2])).show(attributes_level=1)

fft1d
  .help = "Search for the basis vectors of the direct lattice by performing a"
          "series of 1D FFTs along various directions in reciprocal space."
          "This has a lower memory requirement than a single 3D FFT (the fft3d"
          "method). This method may also be more appropriate than a 3D FFT if"
          "the reflections are from narrow wedges of rotation data or from"
          "stills data."
{
}
```
This would also mean that the default `Strategy.phil_help` could be `phil_help = None` which seems cleaner and more Pythonic.